app/src/main/java/com/sthoray/allright/ui/main/viewmodel/MainViewModel.kt

```diff
@@ -35,6 +35,10 @@ class MainViewModel(
         getFeaturedCategories()
         getSecondTierCategories()
     }
+    /** public functions, so we can access it from somewhere else. */
+    fun getRefreshFragment(){
```

Comment:
If I didn't know the purpose of this function in advanced I would need to see what it does. Please try to think of a more descriptive name along with a doc comment to describe the purpose.

Reply:
No worries, i have added some more comments and changed the name for this function. Hopefully it makes sense.